src/main/java/org/jabref/gui/menus/ChangeEntryTypeMenu.java

```diff
@@ -92,4 +102,116 @@ private void populate(ObservableList<MenuItem> items, Collection<BibEntryType> t
     private void populate(Menu menu, Collection<BibEntryType> types, BibEntry entry, UndoManager undoManager) {
         populate(menu.getItems(), types, entry, undoManager);
     }
+
+    //The description is coming from biblatex manual chapter 2
+    //Biblatex documentation is favored over the bibtex,
+    //since bibtex is a subset of biblatex and biblatex is better documented.
+    public static String getDescription(EntryType entryType) {
```

Comment:
Please avoid duplicating the method `getDescription()`, because this makes maintenance hard.